addons/mail/controllers/thread.py

```diff
@@ -191,7 +192,16 @@ def _prepare_message_data(self, post_data, *, thread, **kwargs):
 
     @http.route("/mail/message/post", methods=["POST"], type="jsonrpc", auth="public")
     @add_guest_to_context
-    def mail_message_post(self, thread_model, thread_id, post_data, context=None, **kwargs):
+    def mail_message_post(
+        self,
+        thread_model,
+        thread_id,
+        post_data,
+        context=None,
+        create_pending=False,
+        create_pending_post_delay_seconds=30,
```

Comment:
We don't need 2 parameters, `create_pending` could receive the delay itself (False/0 = send immediately).